src/Microsoft.AspNet.OData.Shared/EdmEntityObject.cs

```diff
@@ -37,7 +40,61 @@ public EdmEntityObject(IEdmEntityTypeReference edmType)
         /// <param name="isNullable">true if this object can be nullable; otherwise, false.</param>
         public EdmEntityObject(IEdmEntityType edmType, bool isNullable)
             : base(edmType, isNullable)
+        {            
+            PersistentInstanceAnnotationsContainer = new ODataInstanceAnnotationContainer();
+            TransientInstanceAnnotationContainer = new ODataInstanceAnnotationContainer();            
+        }
+
+        /// <summary>
+        /// Instance Annotation container to hold Transient Annotations
```

Comment:
```suggestion
        /// Instance annotation container to hold transient annotations.
```

Reply:
wanted those in caps as it is caps in the name and to show significance